django/researchdata/templates/researchdata/list-letter.html

```diff
@@ -21,3 +22,8 @@
     <option value="repository__name">Repository</option>
```

Comment:
Is the double underscore meant to represent a class attribute here? (Just checking that it is not a typo)

Reply:
No, double underscores are used in Django to navigate through relationships. So in this case I want to get the repository name and the field 'repository' is a foreign key field. So I have to choose repository field and then __ to access its fields, e.g. name in this case. So say you have a child and you want to get his grandparent's name, you'd write it as "parent__grandparent__name" ... so this HTML option value will get passed through to Django to deal with in the appropriate format if written like this :)